src/routes/conceptRoutes.js

```diff
@@ -27,7 +28,7 @@ module.exports = function (app) {
 
   app.route(BASE_URL_V1 + '/domains/:domainId/:objectType/search')
     .post(requestMiddleware.createAndValidateRequestBody, requestMiddleware.addChannelFilters,
-      domainService.searchObjectTypeAPI)
+      filterMiddleware.addMetaFilters, domainService.searchObjectTypeAPI)
```

Comment:
channel filters should be merged in with metadata filters

Reply:
will be removed once i complete the meta filter implementation